packages/core-components/src/layout/Sidebar/SidebarSubmenuItem.tsx

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useContext, useState } from 'react';
+import {
+  NavLink,
+  resolvePath,
+  useLocation,
+  useResolvedPath,
+} from 'react-router-dom';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Link from '@material-ui/core/Link';
+import { IconComponent } from '@backstage/core-plugin-api';
+import clsx from 'clsx';
+import { BackstageTheme } from '@backstage/theme';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import ArrowDropUpIcon from '@material-ui/icons/ArrowDropUp';
+import { SidebarItemWithSubmenuContext } from './config';
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  item: {
+    height: 48,
+    width: '100%',
+    '&:hover': {
+      background: '#6f6f6f',
```

Comment:
Hi, is there any chance this component's styles could be made overridable so that it's possible to create a custom theme `BackstageOverrides` for this component?